api/nodes/views.py

```diff
@@ -2935,3 +2937,156 @@ def get_parser_context(self, http_request):
         res = super(LinkedNodesList, self).get_parser_context(http_request)
         res['is_relationship'] = True
         return res
+
+
+class NodeViewOnlyLinksList(JSONAPIBaseView, generics.ListCreateAPIView, NodeMixin):
```

Comment:
We should go ahead and make this filterable on name, anonymous, and date_created.